### PR TITLE
[CP-963] Implementing templates list

### PR DESCRIPTION
**Pull request by igorbezsmertnyi**
Jira: [CP-963]

**Description**
Implement `templates` retrieving and displaying.

<details>
<summary><b>Screenshots</b></summary>
// put images here
</details>

[CP-963]: https://appnroll.atlassian.net/browse/CP-963?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by dkarski — COMMENTED** [review]
The feature will be not visible in daily build. I think that the feature flag should change to `test-alpha-production`.

**Comment by igorbezsmertnyi** [comment]
> The feature will be not visible in daily build. I think that the feature flag should change to test-alpha-production.

Looks like it's already configured in this way https://github.com/mudita/mudita-center/blob/master/packages/app/src/renderer/constants/tab-elements.ts#L56

**Comment by dkarski** [comment]
> > The feature will be not visible in daily build. I think that the feature flag should change to test-alpha-production.
> 
> Looks like it's already configured in this way https://github.com/mudita/mudita-center/blob/master/packages/app/src/renderer/constants/tab-elements.ts#L56

it seems like the value should be changed to `Feature.DisabledOnProduction`. It sounds ridiculous, but _almost_ works (see screenshot).

Zrzut ekranu 2022-05-25 o 14 36 50

File: packages/app/src/renderer/constants/tab-elements.ts [hunk 1]
```diff
@@ -47,13 +47,12 @@ export const tabElements: TabElement[] = [
         label: messages.conversations,
         url: URL_MAIN.messages,
         icon: IconType.Message,
```
Comment (dkarski) [inline packages/app/src/renderer/constants/tab-elements.ts:49]:
```suggestion
        icon: IconType.Message,
        hidden: flags.get(Feature.DisabledOnProduction),
```

Comment (igorbezsmertnyi) [reply packages/app/src/renderer/constants/tab-elements.ts:49]: https://github.com/mudita/mudita-center/pull/735/commits/59bcc410062a4fd7a59e9c02ebe3b7d0d1be69be

**Review by dkarski — APPROVED** [review]

**Pull request merged**